volttrontesting/platform/control_tests/test_vctl_commands.py

```diff
@@ -17,47 +17,47 @@
 
 
 @pytest.mark.control
-def test_needs_connection():
-    # Test command that needs instance running
+def test_no_connection():
+    # Test command that doesn't need instance running.
     p = subprocess.Popen(
-        ["volttron-ctl", "peerlist"],
+        ["volttron-ctl", "list"],
         stdout=subprocess.PIPE,
         stderr=subprocess.PIPE,
     )
     stdout, stderr = p.communicate()
     try:
-        assert "VOLTTRON is not running. This command requires VOLTTRON platform to be running" in stderr.decode(
-            "utf-8")
+        assert "No installed Agents found" in stderr.decode("utf-8")
     except AssertionError:
         assert not stderr.decode("utf-8")
 
 
 @pytest.mark.control
-def test_needs_connection_with_connection():
-    # Verify peerlist command works when instance is running
+def test_needs_connection():
+    # Test command that needs instance running
     p = subprocess.Popen(
         ["volttron-ctl", "peerlist"],
         stdout=subprocess.PIPE,
         stderr=subprocess.PIPE,
     )
     stdout, stderr = p.communicate()
     try:
-        assert "VOLTTRON is not running." in stderr.decode("utf-8")
+        assert "VOLTTRON is not running. This command requires VOLTTRON platform to be running" in stderr.decode(
+            "utf-8")
     except AssertionError:
         assert not stderr.decode("utf-8")
 
 
 @pytest.mark.control
-def test_no_connection():
-    # Test command that doesn't need instance running.
+def test_needs_connection_with_connection():
+    # Verify peerlist command works when instance is running
     p = subprocess.Popen(
-        ["volttron-ctl", "list"],
+        ["volttron-ctl", "peerlist"],
         stdout=subprocess.PIPE,
         stderr=subprocess.PIPE,
     )
     stdout, stderr = p.communicate()
     try:
-        assert "No installed Agents found" in stderr.decode("utf-8")
+        assert "VOLTTRON is not running." in stderr.decode("utf-8")
```

Comment:
This is a same assert as in the previous test case. this test case should test that peerlist is returned successfully